helm/external-dns-app/values.yaml

```diff
@@ -148,6 +148,7 @@ global:
   # dictionary directly into the deployment manifest.
   resources:
     limits:
+      cpu: 100m
```

Comment:
I think we don't want to set cpu limits. Because calculating cpu shares could lead to weird results. There were some fixes upstream and in the kernel? But not sure if present on all the clusters yet. So for now we only set cpu limit when we want the "Guaranteed" QoS class. Then it is equal to the cpu request.

Reply:
Good point! Wasn't aware of this.